src/control/cmd/dmg/pool.go

```diff
@@ -42,8 +42,17 @@ const (
 	msgSizeNoNumber = "size string doesn't specify a number"
 	msgSizeZeroScm  = "non-zero scm size is required"
 	maxNumSvcReps   = 13
+	mibsInMB        = (10 ^ 6) / (2 ^ 20) // convert base2->base10
```

Comment:
I personally find 1000^2 / 1024^2 easier to understand.

Reply:
changed